docker/daemon/client_test.go

```diff
@@ -19,6 +19,8 @@ func TestDockerClientFromNilSystemContext(t *testing.T) {
 
 	assert.Equal(t, dockerclient.DefaultDockerHost, client.DaemonHost(), "The default docker host should have been used")
 	assert.Equal(t, defaultAPIVersion, client.ClientVersion(), "The default api version should have been used")
+
+	assert.Nil(t, client.Close())
```

Comment:
(Absolutely non-blocking: `assert.NoError` results in better output on failure. That can be changed if the test ever fails, so this is fine for now.)